css/font.css

```diff
@@ -0,0 +1,189 @@
+@font-face {
+  font-family: "icomoon";
+  src: url("../fonts/icomoon.eot?efv4m4");
+  src: url("../fonts/icomoon.eot?efv4m4#iefix") format("embedded-opentype"),
+    url("../fonts/icomoon.ttf?efv4m4") format("truetype"),
+    url("../fonts/icomoon.woff?efv4m4") format("woff"),
+    url("../fonts/icomoon.svg?efv4m4#icomoon") format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;500;600;700;800;900;1000&display=swap");
+[class^="icon-"],
+[class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: "icomoon" !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-Add:before {
+  content: "\e91e";
+}
+.icon-arrow-turn-left--down1:before {
+  content: "\e91f";
+  color: #455573;
+}

```

Comment:
remove the duplicate code 

icon-arrow-turn-left--down1

icon-arrow-turn-left--down